f5_openstack_agent/lbaasv2/drivers/bigip/resource_manager.py

```diff
@@ -384,6 +386,45 @@ def _delete_persist_profile(self, bigip, vs):
         self._delete_http_cookie_persist_profile(bigip, vs)
         self._delete_source_addr_persist_profile(bigip, vs)
 
+    def _delete_http_profile(self, bigip, vs):
+        payload = {
+            "name": "http_profile_" + vs['name'],
+            "partition": vs['partition'],
+        }
+        super(ListenerManager, self)._delete(
+            bigip, payload, None, None,
+            helper=self.http_profile_helper)
+
+    def _create_http_profile(self, bigip, vs):
+        # the logic is, if the http_profile_file is configured
```

Comment:
Should we load this file when agent startup? Which is better?

Reply:
> Should we load this file when agent startup? Which is better?

I prefer to load the file when we are trying to use.  By this, the modification will take effect immediately when a new listener is created.  Otherwise, if we load the file at the agent bootup stage, the modification of the file wouldn't take effect until the agent is reloaded?

Reply:
You are right. If it will take effect without restart agent, that will be a big benefit.